intake/catalog/entry.py

```diff
@@ -125,6 +126,32 @@ def plots(self):
         """List custom associated quick-plots """
         return list(self._metadata.get('plots', {}))
 
+    def _ipython_display_(self):
+        """Display the entry as a rich object in an IPython session."""
+        contents, warning = self._display_content()
+        display({ # noqa: F821
+            'application/json': contents,
+            'text/plain': pretty_describe(contents)
+        }, metadata={
+            'application/json': { 'root': self.name }
+        }, raw=True)
+        if warning:
+            display(warning) # noqa: F821
+
+    def _display_content(self):
+        """Create a dictionary with content to display in reprs."""
+        contents = deepcopy(self.describe())
+        warning = None
+        try:
+            contents.update(self.describe_open())
+            if 'plots' in contents['metadata']:
```

Comment:
The removal of the plots dict is specific to the use in the GUI and should just be left in for this case.

Reply:
Okay, I've moved the plots logic back to the GUI.